core/src/HIP/Kokkos_HIP_BlockSize_Deduction.hpp

```diff
@@ -55,6 +55,23 @@
 namespace Kokkos {
 namespace Experimental {
 namespace Impl {
+
+template <typename DriverType, bool, int MaxThreadsPerBlock, int MinBlocksPerSM>
+void iOccupancy(int *numBlocks, int blockSize, int sharedmem) {
+  // FIXME_HIP - currently the "constant" path is ignored, as we always use the
+  //             the local memory launcher
```

Comment:
Can you please add a comment describing the reason for ignoring the `constant` path?